### PR TITLE
Dev 5

### PR DESCRIPTION
**Pull request by ohatsenko**
Added pagination, sorting and OpenAPI

### CODE REVIEW

File: src/main/java/com/example/onlinebookstore/controller/BookController.java [hunk 1]
```diff
@@ -17,35 +20,41 @@
...
     }
 
     @PostMapping
+    @Operation(summary = "Create book", description = "Endpoint for creating a books")
     public BookDto create(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
 
     @PutMapping("/{id}")
+    @Operation(summary = "Update book", description = "Endpoint for updating a books by its ID")
```
Comment (DmytroV95) [inline src/main/java/com/example/onlinebookstore/controller/BookController.java:49]: I think it's better to specify that this method is used to update the book by ID.

File: src/main/java/com/example/onlinebookstore/controller/BookController.java [hunk 2]
```diff
@@ -17,35 +20,41 @@
...
     public BookDto update(@PathVariable Long id,
                           @RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.update(id, bookDto);
     }
 
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
+    @Operation(summary = "Delete book", description = "Endpoint for deleting a books by its ID")
```
Comment (DmytroV95) [inline src/main/java/com/example/onlinebookstore/controller/BookController.java:57]: Same. It's better to specify that this method is used to delete the book by ID.

**Review by DmytroV95 — APPROVED** [review]
GJ! Left some minor comments to improve methods description.

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**